From 84e83b22b01e5d8f7f7b4669eb4376d2e0c00399 Mon Sep 17 00:00:00 2001
From: Thomas HAMEL <thomas-t.hamel@edf.fr>
Date: Fri, 26 Jul 2024 14:09:52 +0200
Subject: [PATCH 1/3] backport cython 0.28.1

Match the version in the python-Cython package from RHEL8
---
 pyproject.toml | 2 +-
 setup.py       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 7d73cf8..89c8a05 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,7 +3,7 @@
 requires = [
     "setuptools>=59.2.0",
     "wheel>=0.37.0",
-    "Cython>=0.29.36",
+    "Cython>=0.28.1",
     "packaging>=21.3"
 ]
 
diff --git a/setup.py b/setup.py
index c2f3077..0b45d63 100644
--- a/setup.py
+++ b/setup.py
@@ -14,7 +14,7 @@ except ImportError:
     from setuptools._vendor.packaging.version import Version
 
 
-CYTHON_VERSION_MIN = "0.29.36" # Keep in sync with pyproject.toml
+CYTHON_VERSION_MIN = "0.28.1" # Keep in sync with pyproject.toml
 SLURM_LIB = "libslurm"
 TOPDIR = Path(__file__).parent
 PYTHON_MIN_REQUIRED = (3, 6)
-- 
2.30.2

